lib/kim/Install.py

```diff
@@ -199,9 +199,9 @@ def geturl(url,fname):
   # download entire kim-api tarball
 
   print("Downloading kim-api tarball ...")
-  geturl(url,"%s/%s.tgz" % (thisdir,version))
+  geturl(url,"%s/%s.txz" % (thisdir,version))
```

Comment:
@ellio167 FYI, this is a problematic change. only rather modern linux systems support .xz compression in tar. using .tgz guarantees the widest compatibility. when considering non-linux system, even general compression support in tar cannot be assumed and one would have to change "tar -xzvf %s.tgz" to "gzip -cd %s.tgz | tar -xvf -". Since the primary target for using Install.py are desktop linux users, the current tar short cut is acceptable.

Reply:
@akohlmey Thanks for the note.  I agree with what you say.  The change is driven by our recent decision to only provide txz files from openkim.org.

Reply:
Ok. so then there is no alternative (unless you change your policy).  I was already about to suggest to try out this change and see if people will report issues.

Reply:
Agreed.  If we get lots of issues, that may prompt (force) us to revisit the policy decision.

Reply:
ok. i've assigned the PR to steve and added a note about compatibility in the general description.